lib/segment/src/index/field_index/numeric_index/mod.rs

```diff
@@ -254,7 +269,7 @@ impl<T: Encodable + Numericable + Default> NumericIndexInner<T> {
         let range = match range {
             RangeInterface::Float(float_range) => float_range.map(T::from_f64),
             RangeInterface::DateTime(datetime_range) => {
-                datetime_range.map(|dt| T::from_i64(dt.timestamp()))
+                datetime_range.map(|dt| T::from_i128(i128::from(dt.timestamp())))
```

Comment:
why DateTime index is affected by this change?

Reply:
Because I updated [this](https://github.com/qdrant/qdrant/pull/4738/files#diff-5adcfd0e240fb6450bb128fad52ac82295ebb05f83cb4c55c05de70593e69db9R57) trait function in order to be able to pass UUIDs. The DateTime change is just to make it compatible with the updated trait.